packages/components/src/Avatar/docs/assets/index.js

```diff
@@ -0,0 +1,4 @@
+import ChrisHadfield from "./hadfield.png";
```

Comment:
you should use another image. Hadfield was used in the stories because it was a space themed DS. We should find something else for hopper

Reply:
do we need a specific theme? I was just using anything.. For the new image I added, I took it from anime. I have no idea what else to use.